packages/javascript/src/i18n/hi-IN.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* eslint-disable sort-keys */
+
+import { I18nTranslations, I18nMetadata, I18nBundle } from "../models/i18n";
+
+const translations: I18nTranslations = {
+  /* |---------------------------------------------------------------| */
+  /* |                        Elements                               | */
+  /* |---------------------------------------------------------------| */
+
+  /* Buttons */
+  "elements.buttons.signIn": "साइन इन",
```

Comment:
We use single quotes.
As mentioned in https://github.com/asgardeo/javascript/pull/183#issuecomment-3347829537, these can be auto-fixed.
```suggestion
  'elements.buttons.signIn': 'साइन इन',
```